pkg/config/utils/inventory.go

```diff
@@ -0,0 +1,25 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package utils
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// InventoryEnabled returs true if 'enable_metadata_collection' and 'inventories_enabled' are set to true in the
+// configuration.
+func InventoryEnabled(conf config.Reader) bool {
```

Comment:
As this function is used only in `comp/metadata` what do you think about moving the function inside a subfolder of `comp/metadata`?